tests/baselines/bert_large_uncased_whole_word_masking.json

```diff
@@ -32,19 +32,19 @@
             "single_card": {
                 "learning_rate": 3e-5,
                 "train_batch_size": 32,
-                "eval_f1": 0.9005,
-                "train_runtime": 115.5984,
-                "train_samples_per_second": 169.218,
+                "eval_f1": 0.91,
+                "train_runtime": 114.2319,
+                "train_samples_per_second": 169.303,
                 "extra_arguments": [
                     "--max_seq_length 128"
                 ]
             },
             "multi_card": {
                 "learning_rate": 3e-5,
                 "train_batch_size": 16,
-                "eval_f1": 0.8885,
-                "train_runtime": 189.1009,
-                "train_samples_per_second": 702.962,
+                "eval_f1": 0.8841,
+                "train_runtime": 77.2454,
+                "train_samples_per_second": 702.284,
```

Comment:
here has no change to train sample per sec while train time reduced to more than half